applications/maplab-node/launch/maplab_node_client_setup.launch

```diff
@@ -17,6 +17,7 @@
   <arg name="maplab_use_clahe" default="true"/>
   <arg name="maplab_map_directory" default="/tmp/maplab_node/$(arg maplab_robot_name)"/>
   <arg name="maplab_map_add_odometry_edges_if_less_than_n_common_landmarks" default="20"/>
+  <arg name="maplab_use_unique_ts_prefix" default="false"/>
```

Comment:
Just writing timestamp in full, instead of abbreviating it as ts, would have been nicer. Aside from that it looks good to me.